packages/services/src/competition.service.ts

```diff
@@ -1516,7 +1496,10 @@ export class CompetitionService {
 
     // Use bulk portfolio value calculation
     const portfolioValues =
-      await this.tradeSimulatorService.calculateBulkPortfolioValues(agents);
+      await this.tradeSimulatorService.calculateBulkPortfolioValues(
+        agents,
+        competitionId,
```

Comment:
Nice! This will resolve the last item in https://github.com/recallnet/js-recall/pull/1522